ci/infra/testrunner/README.md

```diff
@@ -1,63 +1,213 @@
+
+Example:
 ```
-SKUBA_BINPATH=/path/to/bin/skuba
-SKUBA_SRCPATH=/path/to/src/skuba
+vmware: 
+   env_file: "/path/to/env/file"
 ```
 
-or
+### Skuba
+
+The Skuba section defines the location and execution options for the `skuba` command. As `testrunner` can be used either from a local development or testing environment or a CI pipeline, the configuration allows to define the location of both the source and the binary. Please notice that the source location is used as default location for other configuration elements, such as terraform files, even if the `skuba` binary is specified. 
+
+* binpath: path to skuba binary
+* srcpath: path to skuba source. Used to locate other resources, like terraform configuration, and ssh keys.
+* verbosity: verbosity level for skuba command execution
 
 ```
 skuba:
-  binpath: "" # path to skuba binary
-  srcpath: "" # path to skuba source
+  binpath: "/usr/bin/"
+  srcpath: "/go/src/github.com/SUSE/skuba"
+  verbosity: 5
+```
+
+### Log
+
+Testrunner sends output to both a console and file logger handlers, configured using the following `log` variables:
+
+* file: name of the file used to send a copy of the log with verbosity `DEBUG`. This file is located under the `workspace` directory.
+* level: debug verbosity level to console. Can be any of `DEBUG`, `INFO`, `WARNING`, `ERROR`. Defaults to `INFO`.
+* overwrite: boolean that indicates if the content of the log file must be overwritten (`True`) or log entries must be appended at the end of the file if it exists. Defaults to `False` (do not overwrite) 
+* quiet: boolean that indicates if `testrunner` will send any output to console (`False`) or not will execute silently (`True`). Quiet mode is useful when `testrunner` is used as a library. Defaults to `False`.
+
+Example:
+```
+log:
+  level: DEBUG
+```
+
+### Test
+
+* no_destroy: boolean that indicates if provisioned resources should be deleted when test ends. Defaults to `False`
+
+```
+no_destroy: True  #keep resources after test ends
+```
+
+## Environment Setup
+
+This section details how to setup `testrunner` 
+
+### Local setup
+
+Copy `vars.yaml` to `/path/to/myvars.yaml`, set the variables according to your environment and needs, and use the `--vars /path/to/myvars.yaml` CLI argument when running `testrunner`.
+
+#### Work Environment
+
+`testrunner` requires a working directory, which is specified in the `workspace` configuration parameter. The content of this directory can be erased or overwritten by `testrunner`. Be sure you create a directory to be used as workspace which is not located under your local working copy of the `skuba` project.
+
+
+```
+workspace: "/path/to/workspace"
+username:  "my-user-test"
+```
+
+
+#### skuba and platform
+
+Set the `skuba` and `terraform parameters depending on how you are testing `skuba`: 
+* If testing from local source:
+```
+  skuba:
+    srcpath: "/path/to/local/skuba/repo"
+    binpath: "path/to/go/bin/directory"
```

Comment:
```suggestion
    binpath: "/path/to/go/bin/directory"
```
Should this be an absolute path?  If relative is supported, that should be noted somewhere.